### ISSUE TITLE
Remove @room tag

### ISSUE THREAD

**Issue by chagai95** [anoadragon453/matrix-reminder-bot#80]
Feature request to just support regular reminders without tagging everyone. Would also be useful to use with double puppeted bridges without having to implement this on every bridge...

**Comment by HarHarLinks** [comment]
are you proposing to add the `!remindme` command which already exists?

**Comment by chagai95** [comment]
no, that command only reminds you if I understand correctly, I would like to remind everyone but without tagging everyone, think of it as more of a send messages later feature, like telegram has, would be nice to time messages without tagging the whole room or in direct messages without tagging the room when there is only one other person in it anyway, maybe call it !message or !sendlater

**Comment by anoadragon453** [comment]
Do you want a command that won't tag you *or* the room? I'm confused why `!remindme` doesn't solve your problem, as it does not use `@room`.

**Comment by chagai95** [comment]
Exactly I would like a command that does not tag me or the room, you think that's something that could interest everyone? I could just try adding it myself otherwise..

**Comment by anoadragon453** [comment]
I think it could be useful, yes. If you as an admin want to set up a weekly broadcast for a room without specifically attaching your name to the resulting message.

I'd like it as a new command though. Some suggestions for names might be `!remindgeneral`, `!remindbroadcast`, `!remindanyone`?

**Comment by chagai95** [comment]
maybe a new command? message?

**Comment by anoadragon453** [comment]
As in `!message` or `!remindmessage`? Neither sound too intuitive to me.

**Comment by chagai95** [comment]
Maybe !sendlater ?
Or !sendat ?

!sendat 5 am tomorrow; wake up

**Comment by chagai95** [comment]
Any progress here? Can I help? 🙈

**Comment by anoadragon453** [comment]
I think this is still stuck in the design phase - but how about just `!reminder`?

**Comment by chagai95** [comment]
Sounds great! I think this'll really upgrade how useful this bot can be 😉 sending messages at later time period is pretty cool

**Comment by HarHarLinks** [comment]
If `!reminder` is happening, might I request that we get an option to make it a `m.notice` when firing? Since this is not a pinging reminder (unless message text includes a ping), that would be viable.

**Comment by chagai95** [comment]
So it won't trigger a notification on all clients? What's the point of the reminder then? @HarHarLinks 🙈

**Comment by HarHarLinks** [comment]
If it isn't time-critical, but more read "at your leisure"? Arguably if you want to trigger a notification, you would use `!remindroom`...
